schemas/registry-completions.schema.json

```diff
@@ -67,16 +69,25 @@
             "path"
           ]
         },
+        "documentation": {
+          "type": "string",
+          "description": "An optional \"endpoint\" to call to provide documentation for specified variable, which can be displayed to the client in the response. This can provide description information about the item. The value should contain the variable name in the template. Baseless URLs can be supplied and the host for the configuration file will be used as the base.",
+          "examples": [
+            "https://example.com/api/documentation/packages/${package}/${{version}}/${path}",
+            "/api/documentation/packages/${package}/${{version}}/${path}"
+          ]
+        },
         "url": {
           "type": "string",
-          "description": "The \"endpoint\" to call to provide values to complete the specifier. This endpoint should return an array of strings. Parsed values can be substituted using ${key} syntax.",
+          "description": "The \"endpoint\" to call to provide values to complete the specifier. This endpoint should return an array of strings. Parsed values can be substituted using ${key} syntax. In order to support incremental completions, the value should contain the variable name in the template. Baseless URLs can be supplied and the host for the configuration file will be used as the base.",
           "examples": [
             "https://example.com/api/packages",
-            "https://example.com/api/packages/{package}",
-            "https://example.com/api/packages/{package}/${{version}}"
+            "https://example.com/api/packages/${package}",
+            "https://example.com/api/packages/${package}/${{version}}",
+            "/api/packages/${package}/${{version}}/${path}"
           ]
         }
       }
     }
   }
-}
+}
```

Comment:
I wonder what is removing the final newlines. Not that it really matters, but seems like we don't have code formatting on this project?

Reply:
Having an old formatter configured in my IDE for JSON files and not running `npm fmt` before comitting!